x/dex/types/message_place_limit_order_test.go

```diff
@@ -119,6 +163,40 @@ func TestMsgPlaceLimitOrder_ValidateBasic(t *testing.T) {
 				AmountIn:         math.OneInt(),
 			},
 		},
+
+		{
+			name: "valid msg price",
```

Comment:
"valid msg price" and "valid msg price > 1" have the same test arguments, is it intended to be?